xgi/algorithms/connected.py

```diff
@@ -35,11 +35,12 @@ def is_connected(H):
 
     Example
     -------
-        >>> import xgi
-        >>> n = 1000
-        >>> ps = [0.01, 0.001]
-        >>> H = xgi.random_hypergraph(n, ps)
-        >>> print(xgi.is_connected(H))
+    >>> import xgi
+    >>> n = 1000
+    >>> ps = [0.01, 0.001]
+    >>> H = xgi.random_hypergraph(n, ps)
```

Comment:
Wondering if we want to use the seed option that @maximelucas implemented so we can assure consistent results.